pasqal_cloud/__init__.py

```diff
@@ -44,18 +45,34 @@ def __init__(
         To authenticate to PASQAL Cloud, you have to provide either an
         email/password combination or a TokenProvider instance.
         You may omit the password, you will then be prompted to enter one.
+
+        Also, you can use the parameter `env` to have the endpoints and auth0 endpoints filled automatically.
         """
 
         # Ticket (#622), to be removed, used to avoid a breaking change during the group to project renaming
         if not project_id:
             if not group_id:
-                raise TypeError("Either a group_id or project_id has to be given as argument")
-            warn('The parameter group_id is deprecated, from now use project_id instead',
+                raise TypeError("Either a 'group_id' or 'project_id' has to be given as argument")
+            warn("The parameter 'group_id' is deprecated, from now on use 'project_id' instead",
                  DeprecationWarning,
                  stacklevel=2
                  )
             project_id = group_id
 
+        # TBR Ticket slot
+        if endpoints or auth0:
+            warn("The parameters 'endpoints' and 'auth0' are deprecated, from now use 'env' instead",
+                 DeprecationWarning,
+                 stacklevel=2
+                 )
+
+        if env:
+            # TBR Ticket slot
+            if endpoints or auth0:
+                raise TypeError("Parameters 'endpoints' and 'auth0' must be used OR 'env' alone otherwise, not all "
```

Comment:
Suggestion: The `env` argument is incompatible with `endpoints` and `auth0` arguments.  You may either target a pre-defined environment, or define custom auth0 configuration and API endpoints.